source/features/prevent-pr-commit-link-loss.tsx

```diff
@@ -1,16 +1,33 @@
-import delegate from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
+import AlertIcon from 'octicon/alert.svg';
+import delegate from 'delegate-it';
+import React from 'dom-chef';
+import select from 'select-dom';
 
 import features from '../libs/features';
-import {preventPrCommitLinkBreak} from '../libs/utils';
+import {containsPrCommitLink, preventPrCommitLinkBreak} from '../libs/utils';
 
 function init(): void {
-	delegate(document, 'form#new_issue, form.js-new-comment-form', 'submit', updateTextArea);
+	delegate(document, 'form#new_issue textarea, form.js-new-comment-form textarea', 'input', handleTextAreaChange);
 }
 
-function updateTextArea(event: delegate.Event): void {
-	const field = event.delegateTarget.querySelector('textarea')!;
-	field.value = preventPrCommitLinkBreak(field.value);
+function handleTextAreaChange(event: delegate.Event): void {
```

Comment:
You can set this and avoid the `as`sertion later
```suggestion
function handleTextAreaChange(event: delegate.Event<InputEvent, HTMLTextAreaElement>): void {
```